primitives/src/network.rs

```diff
@@ -0,0 +1,37 @@
+use codec::{Decode, Encode};
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_runtime::RuntimeDebug;
+
+/// Network type for parallel.
+#[derive(PartialEq, Eq, Clone, Copy, Encode, Decode, RuntimeDebug)]
```

Comment:
Encode, decode, Eq, should not be needed, only add the derived implementation that needed.

Reply:
Fix it in next commit.